zipline/utils/calendars/trading_calendar.py

```diff
@@ -645,15 +725,15 @@ def _special_dates(self, calendars, ad_hoc_dates, start_date, end_date):
 
     def _special_opens(self, start, end):
         return self._special_dates(
-            self.special_opens_calendars,
+            self.special_opens,
```

Comment:
I renamed `_special_opens` to `_calculate_special_opens`, etc.  Hope that helps.